Google.Api.Gax.Grpc/RetrySettings.cs

```diff
@@ -53,64 +59,37 @@ public sealed class RetrySettings
         /// and returns an actual delay which is a uniformly random value between 0 and the maximum. This
         /// is good enough for most applications, but makes precise testing difficult.
         /// </remarks>
-        public IJitter DelayJitter { get; }
-
-        /// <summary>
-        /// Constructs an instance with the given backoff configuration, the default RPC filter and
-        /// jitter.
-        /// </summary>
-        /// <param name="retryBackoff">The backoff policy for the time between retries. Must not be null.</param>
-        /// <param name="timeoutBackoff">The backoff policy for timeouts of retries. Must not be null.</param>
-        /// <param name="totalExpiration">The total expiration, across all retries. Must not be null.</param>
-        public RetrySettings(
-            BackoffSettings retryBackoff,
-            BackoffSettings timeoutBackoff,
-            Expiration totalExpiration) : this(retryBackoff, timeoutBackoff, totalExpiration, null, null)
-        {
-        }
-
-        /// <summary>
-        /// Constructs an instance with the given configuration, and the default jitter.
-        /// </summary>
-        /// <param name="retryBackoff">The backoff policy for the time between retries. Must not be null.</param>
-        /// <param name="timeoutBackoff">The backoff policy for timeouts of retries. Must not be null.</param>
-        /// <param name="totalExpiration">The total expiration, across all retries. Must not be null.</param>
-        /// <param name="retryFilter">A predicate to determine whether or not a particular exception should cause the operation to be retried,
-        /// or null for the default filter.</param>
-        public RetrySettings(
-            BackoffSettings retryBackoff,
-            BackoffSettings timeoutBackoff,
-            Expiration totalExpiration,
-            Predicate<RpcException> retryFilter) : this(retryBackoff, timeoutBackoff, totalExpiration, retryFilter, null)
-        {
-        }
+        public IJitter BackoffJitter { get; }
 
         /// <summary>
-        /// Constructs an instance with the given configuration.
+        /// FIXME
```

Comment:
Fix ;)

Reply:
Have kept this internal, but added static factory methods.